### PR TITLE
feat: hit ceramic healthcheck endpoint

### PR DESCRIPTION
**Pull request by smrz2001**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ build: release
 
 .PHONY: release
 release:
-	$(eval BIN=$(shell sh -c "RUSTFLAGS='-D warnings' cargo test --no-run --locked --release -q --message-format=json | jq -r 'select(.executable != null) | .executable'"))
```
Comment (smrz2001) [inline Makefile:15]: Removed flag for warnings because there's code used only in tests that the build keeps complaining about.

Comment (nathanielc) [reply Makefile:15]: Would it make sense to use a `#[allow(dead_code)]` attribute instead of ignoring all warnings?

**Review by nathanielc — APPROVED** [review]
LGTM with a few small questions.

File: Cargo.toml [hunk 2]
```diff
@@ -4,7 +4,11 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+async-once-cell = "0.5.3"
```
[the remark this answers is not in the document]
Comment (smrz2001) [reply Cargo.toml:7]: No, we did not. Removing this one and another unused dep.

**Pull request merged**